### PR TITLE
Update learn.yaml with 32 hour course

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
## Bug

- [x] Make sure the linting passes
- [x] Related issues linked using `fixes #number`
- [x] The new feature, or the fix is tested with jest and if applicable with cypress
- [x] The code is properly formatted using the project's prettier and eslint settings
- [x] Make sure that everything is properly typed in the code.
- [x] Contribution guidelines are met

## Feature

- [x] Make sure the linting passes
- [x] Implements an existing feature request or change request. Make sure the feature request has been accepted for
      implementation before opening a PR.
- [x] Related issues linked using `fixes #number`
- [x] The new feature, or the fix is tested with jest and if applicable with cypress
- [x] The code is properly formatted using the project's prettier and eslint settings
- [x] Make sure that everything is properly typed in the code.
- [x] Contribution guidelines are met
- [x] Documentation added

## Resource

- [x] Make sure the linting passes
- [x] Added the data in the right `yaml` file

## Article

- [x] Make sure the linting passes
- [x] The article includes an author
- [x] the article includes an author thubnail
- [x] The article has been tested locally and it works
- [x] The images have been added to the public folder

### CODE REVIEW

**Comment by PatrickAlphaC** [comment]
Fixes https://github.com/smartcontractkit/blockchain-developer-hub/issues/356

**Comment by PatrickAlphaC** [comment]
I think everything passes... let me know if not

**Comment by danielgruesso** [comment]
Thanks @PatrickAlphaC. Seems build is failing in L10-14; the indentation is off by a space. Also t in title and a in author should be on the same column. I can't update your fork unfortunately.

File: data/learn.yaml [hunk 1]
```diff
@@ -6,6 +6,13 @@ resources:
       href: '/learn#evm-chains-courses'
       overview: "Introductory short form content to help you get up and running quickly. Deploy contracts and see them in action."
       data:
+       - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
+        author: FreeCodeCamp
+        image: "https://github.com/smartcontractkit/full-blockchain-solidity-course-js/blob/main/img/blockchain1.png?raw=true"
+        level: Beginner
+        href: https://www.youtube.com/watch?v=gyMwXuJrbJQ
+        description: The \#1 resource for learning smart contracts & professional blockchain development. From zero experience to master.
```
Comment (danielgruesso) [inline data/learn.yaml:14]: ``
```suggestion
       - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
        author: FreeCodeCamp
        image: "https://github.com/smartcontractkit/full-blockchain-solidity-course-js/blob/main/img/blockchain1.png?raw=true"
        level: Beginner
        href: https://www.youtube.com/watch?v=gyMwXuJrbJQ
        description: The \#1 resource for learning smart contracts & professional blockchain development. From zero experience to master.
```

File: data/learn.yaml [hunk 2]
```diff
@@ -6,6 +6,13 @@ resources:
       href: '/learn#evm-chains-courses'
       overview: "Introductory short form content to help you get up and running quickly. Deploy contracts and see them in action."
       data:
+       - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
```
Comment (danielgruesso) [inline data/learn.yaml:9]:
```suggestion
      - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"```

File: data/learn.yaml [hunk 3]
```diff
@@ -6,6 +6,13 @@ resources:
       href: '/learn#evm-chains-courses'
       overview: "Introductory short form content to help you get up and running quickly. Deploy contracts and see them in action."
       data:
+      - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"```
```
Comment (danielgruesso) [inline data/learn.yaml:9]:
```suggestion
      - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
```

File: data/learn.yaml [hunk 4]
```diff
@@ -6,6 +6,13 @@ resources:
       href: '/learn#evm-chains-courses'
       overview: "Introductory short form content to help you get up and running quickly. Deploy contracts and see them in action."
       data:
+      - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
+        author: FreeCodeCamp
+        image: "https://github.com/smartcontractkit/full-blockchain-solidity-course-js/blob/main/img/blockchain1.png?raw=true"
+        level: Beginner
+        href: https://www.youtube.com/watch?v=gyMwXuJrbJQ
+        description: The \#1 resource for learning smart contracts & professional blockchain development. From zero experience to master.
```
Comment (danielgruesso) [inline data/learn.yaml:14]:
```suggestion
        description: The #1 resource for learning smart contracts & professional blockchain development. From zero experience to master.
```

File: data/learn.yaml [hunk 5]
```diff
@@ -6,6 +6,13 @@ resources:
       href: '/learn#evm-chains-courses'
       overview: "Introductory short form content to help you get up and running quickly. Deploy contracts and see them in action."
       data:
+      - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
+        author: FreeCodeCamp
+        image: "https://github.com/smartcontractkit/full-blockchain-solidity-course-js/blob/main/img/blockchain1.png?raw=true"
+        level: Beginner
+        href: https://www.youtube.com/watch?v=gyMwXuJrbJQ
+        description: The #1 resource for learning smart contracts & professional blockchain development. From zero experience to master.
```
Comment (danielgruesso) [inline data/learn.yaml:14]:
```suggestion
        description: The "#1" resource for learning smart contracts & professional blockchain development. From zero experience to master.
```

File: data/learn.yaml [hunk 6]
```diff
@@ -6,6 +6,13 @@ resources:
       href: '/learn#evm-chains-courses'
       overview: "Introductory short form content to help you get up and running quickly. Deploy contracts and see them in action."
       data:
+      - title: "Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course"
+        author: FreeCodeCamp
+        image: "https://github.com/smartcontractkit/full-blockchain-solidity-course-js/blob/main/img/blockchain1.png?raw=true"
+        level: Beginner
+        href: https://www.youtube.com/watch?v=gyMwXuJrbJQ
+        description: The "#1" resource for learning smart contracts & professional blockchain development. From zero experience to master.
```
Comment (danielgruesso) [inline data/learn.yaml:14]:
```suggestion
        description: "The #1 resource for learning smart contracts & professional blockchain development. From zero experience to master."
```

**Pull request merged**